app_flutter/lib/state/flutter_build.dart

```diff
@@ -233,14 +233,12 @@ class FlutterBuildState extends ChangeNotifier {
     List<CommitStatus> statuses,
     CommitStatus statusToFind,
   ) {
-    for (int index = 0; index < statuses.length; index++) {
+    for (int index = 0; index < statuses.length; index += 1) {
```

Comment:
Is this left over from debugging?

Reply:
https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#prefer-over :-)